web/ViewModels/PropertyViewModel.cs

```diff
@@ -1,17 +1,56 @@
-﻿using System.ComponentModel.DataAnnotations;
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
 using BricksAndHearts.Database;
 
 namespace BricksAndHearts.ViewModels;
 
 public class PropertyViewModel
 {
-    [Required] public string Address { get; set; } = string.Empty;
+    [Required] public PropertyAddress Address { get; set; }
+
+    [Required] public string PropertyType { get; set; } = string.Empty;
```

Comment:
I haven't generally been asking for this, but it's good practice to put sensible limits on field size by default, e.g. `[StringLength(10000)]`. They should be much higher than could ever be reasonable to actually use.

It's not a big deal, but it makes it harder to do denial of service attacks, e.g. no-one needs to be able to add a 100 million character address line and that might just break the website.